### PR TITLE
The Ember Times No. 171 - January 15th 2021

### PR DESCRIPTION
**Pull request by amyrlam**
❄️ New to The Ember Times? If you want to claim a topic to write about, please comment here on the PR or chat with us in [#support-ember-times](https://discordapp.com/channels/480462759797063690/485450546887786506) on the [Ember Discord](https://discord.gg/emberjs)! ❄️

***

### Addon/App releases

- [ ] [ember-local-storage-decorator](https://twitter.com/jelhan1/status/1341440143899680777)
- [ ] [Beta release of ember-template lint v3](https://twitter.com/melaniersumner/status/1349820518216708096)

### Articles

- [x] Writeup on new blog.emberjs.com? #st-website PR is opened:  https://github.com/ember-learn/ember-blog/pull/876 (🔒 @jenweber)
- [x] [Understanding `args` in Glimmer components](https://v5.chriskrycho.com/journal/understanding-args-in-glimmer-components/) (🔒 @chrisrng )
- [x] [Ember 3.24 Released](https://blog.emberjs.com/ember-3-24-released/) (🔒 @chrisrng )
- [x] [EmberConf 2021 CfP Open](https://cfp.emberconf.com/events/emberconf-2021)  (🔒 @amyrlam )
- [x] [Micro-Frontends with Glimmer.js and single-spa](https://dev.to/rajasegar/micro-frontends-with-glimmer-js-and-single-spa-42aa) (:lock: @jaredgalanis)
- [ ] [Storybook for Ember tutorial](https://twitter.com/storybookjs/status/1343663019717513225)  (🔒 @amyrlam ping gossi on discord)
  - https://twitter.com/unistyler/status/1343853626901590016
  - https://gos.si/blog/ember-with-storybook-behind-the-scenes/
- [ ] [Getting Started With TypeScript In Your EmberJS App](https://twitter.com/knownasilya/status/1349571457983127561) (🔒 @amyrlam ping fozy81 on discord )
- [ ] Review and input welcome here: I just finished a significant revision to the Typed Ember RFC: Defining Semantic Versioning for Published Types—https://github.com/typed-ember/ember-cli-typescript/pull/1158 (which is groundwork for what we need for TS support in Ember itself!).
- [ ] Hi! At Mimiquate (https://www.mimiquate.com/) we have just released Timo (https://timo.mimiquate.xyz/), an Ember web app to simplify virtual meeting scheduling on these remote first times. It's open source: https://github.com/mimiquate/Timo.
  - Here is an article telling the story of its inception, hope you like it :)
  - https://www.linkedin.com/posts/juan-azambuja-62b4a872_elixir-virtualmeetings-webdevelopment-activity-6739527429657784320-0MYV
- [ ] [Building Prototypes with Ember.js](https://twitter.com/simplabs/status/1338871240342327296)
- [ ] [Building a Router component for Glimmer.js](https://dev.to/rajasegar/building-a-router-component-for-glimmer-js-52bk)
- [ ] [Setup GitHub Actions as CI for Ember Addons](https://twitter.com/jelhan1/status/1346034576683130880)
- [ ] [I ❤ Ember.js — Introduction](https://twitter.com/krefsar/status/1346828521495257092)
  - https://emart.medium.com/i-ember-js-routes-ember-data-b6baf4308803
- [ ] [Ember.js tutorial for beginners in 2020](https://www.youtube.com/channel/UCQLEPbkEmwz4mABAV-kzQTQ) (video series, first mentioned in Ember Weekly)
- [ ] [UI Automation Testing with Ember.js](https://twitter.com/GirlGeekX/status/1346924838536048641) (video)
- [ ] [Track performance of your web application with JavaScript](https://twitter.com/AfkZiad/status/1349724640969539585)
- [ ] [Trapping Focus in a Modal in Ember](https://davidtang.io/trapping-focus-in-a-modal-in-ember/)

### Readers' Questions / Miscellaneous

None

### RFCs

- [x] [Deprecate `{{hasBlock}}` in templates](https://github.com/emberjs/rfcs/pull/689)  (🔒 @ijlee2)
- [x] [Deprecate `{{attrs}}` in templates](https://github.com/emberjs/rfcs/pull/690)
- [x] [Deprecate passing `classBinding` and `classNameBindings` as arguments](https://github.com/emberjs/rfcs/pull/691)
- [x] [Deprecate Array Observers](https://github.com/emberjs/rfcs/pull/692)
- [x] [Deprecate `<LinkTo>` Component Positional Arguments](https://github.com/emberjs/rfcs/pull/698)
- [x] [Deprecate Octane Optional Features](https://github.com/emberjs/rfcs/pull/704)
- [x] [Deprecate jQuery Integration Optional Feature](https://github.com/emberjs/rfcs/pull/705)
- [x] [Deprecate the Ember Global](https://github.com/emberjs/rfcs/pull/706)
- [x] [Modernize Built-In Components (Part 2)](https://github.com/emberjs/rfcs/pull/707)
- [ ] [Argument Validation Primitives](https://github.com/emberjs/rfcs/pull/694)
- [ ] [Argument Default Primitives](https://github.com/emberjs/rfcs/pull/695)
- [ ] [Replace Travis CI with GitHub Actions in generated Ember CLI projects](https://github.com/emberjs/rfcs/pull/696)

## 📆  Future stories

- [ ] "I created one large issue with almost all ember-related articles across last 12 months (https://github.com/ember-community-russia/awesome-ember/issues/56), if smb has capacity to land some part of it into awesome list or add more articles into ticket - it will be great!" from https://discord.com/channels/480462759797063690/480777444203429888/770366277083463702

## 🤔  Questions?

See #support-ember-times on Discord or CONTRIBUTING.

## ✅  Publishing TODOs:

- [ ] Ensure no use of Markdown italic, use `<span style="font-style: italic;">` instead 
- [ ] Check Netlify preview e.g. buttons, to ensure display on emberjs.com/blog is good
- [ ] Crosspost on dev.to, delete any alex, redo any span italics, manually redo Readers Question picture, add Canonical URL 
- [ ] Set up tweet for Monday morning on tweetdeck
- [ ] Post $natural for #news-and-announcements in Discord

### CODE REVIEW

File: content/the-ember-times-issue-171.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+---
+title: The Ember Times - Issue No. 171
+author:
+  - the-crowd # replace with real authors from the author folder (add yourself if you're not there)
+date: 2021-01-15T00:00:00.000Z
+tags:
+  - newsletter
+  - ember-js-times
```
Comment (ijlee2) [inline content/the-ember-times-issue-171.md]: @amyrlam It looks like we don't need to use `ember-js-times` tag anymore. The test in CI failed with the message,

```
You have defined a post with tag "ember-js-times" but there is no tag with that id. To create this tag run 'npx ember g tag ember-js-times'
```

Let's update the template in a separate pull request, then rebase this branch?

https://raw.githubusercontent.com/ember-learn/ember-blog/master/post-templates/embertimes-template.md

Comment (ijlee2) [reply content/the-ember-times-issue-171.md]: Ah, you made a [PR](https://github.com/ember-learn/ember-blog/pull/849) already. 🙂

Comment (amyrlam) [reply content/the-ember-times-issue-171.md]: 👍 removed it here!

**Comment by amyrlam** [comment]
P.S. @ijlee2 added new 2021 tag to https://github.com/ember-learn/ember-blog/pull/849

**Comment by gossi** [comment]
I also have a follow-up article explaining storybook and ember integration details:

- https://twitter.com/unistyler/status/1345690746523955200
- https://gos.si/blog/ember-with-storybook-behind-the-scenes/

File: content/the-ember-times-issue-171.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Contributors' corner 👏](https://guides.emberjs.com/release/contributing/repositories/)
+
+<p>This week we'd like to thank our siblings for their contributions to Ember and related repositories! 💖</p>
```
Comment (ijlee2) [inline content/the-ember-times-issue-171.md]:
```suggestion
<p>This week we'd like to thank <a href="https://github.com/MelSumner" rel="noopener noreferrer" target="_blank">Melanie Sumner (@MelSumner)</a>, <a href="https://github.com/chadhietala" rel="noopener noreferrer" target="_blank">Chad Hietala (@chadhietala)</a>, <a href="https://github.com/chiragpat" rel="noopener noreferrer" target="_blank">Chirag Patel (@chiragpat)</a>, <a href="https://github.com/chrisrng" rel="noopener noreferrer" target="_blank">Chris Ng (@chrisrng)</a>, <a href="https://github.com/snewcomer" rel="noopener noreferrer" target="_blank">Scott Newcomer (@snewcomer)</a>, <a href="https://github.com/fozy81" rel="noopener noreferrer" target="_blank">Tim (@fozy81)</a>, <a href="https://github.com/dwickern" rel="noopener noreferrer" target="_blank">Derek Wickern (@dwickern)</a>, <a href="https://github.com/ijlee2" rel="noopener noreferrer" target="_blank">Isaac Lee (@ijlee2)</a>, <a href="https://github.com/knownasilya" rel="noopener noreferrer" target="_blank">Ilya Radchenko (@knownasilya)</a>, <a href="https://github.com/kategengler" rel="noopener noreferrer" target="_blank">Katie Gengler (@kategengler)</a>, <a href="https://github.com/bmish" rel="noopener noreferrer" target="_blank">Bryan Mishkin (@bmish)</a>, <a href="https://github.com/jenweber" rel="noopener noreferrer" target="_blank">Jen Weber (@jenweber)</a>, <a href="https://github.com/jeffdaley" rel="noopener noreferrer" target="_blank">Jeff Daley (@jeffdaley)</a>, <a href="https://github.com/jaredgalanis" rel="noopener noreferrer" target="_blank">Jared Galanis (@jaredgalanis)</a>, <a href="https://github.com/jderr-mx" rel="noopener noreferrer" target="_blank">John Derr (@jderr-mx)</a>, <a href="https://github.com/locks" rel="noopener noreferrer" target="_blank">Ricardo Mendes (@locks)</a>, <a href="https://github.com/dcyriller" rel="noopener noreferrer" target="_blank">Cyrille David (@dcyriller)</a>, <a href="https://github.com/rwjblue" rel="noopener noreferrer" target="_blank">Robert Jackson (@rwjblue)</a>, <a href="https://github.com/czikarito" rel="noopener noreferrer" target="_blank">Pawel Kuwik (@czikarito)</a>, <a href="https://github.com/nickschot" rel="noopener noreferrer" target="_blank">Nick Schot (@nickschot)</a>, <a href="https://github.com/Alonski" rel="noopener noreferrer" target="_blank">Alon Bukai (@Alonski)</a>, <a href="https://github.com/pzuraq" rel="noopener noreferrer" target="_blank">Chris Garrett (@pzuraq)</a>, <a href="https://github.com/sandstrom" rel="noopener noreferrer" target="_blank">@sandstrom</a>, <a href="https://github.com/rwwagner90" rel="noopener noreferrer" target="_blank">Robert Wagner (@rwwagner90)</a>, <a href="https://github.com/kennethlarsen" rel="noopener noreferrer" target="_blank">Kenneth Larsen (@kennethlarsen)</a>, <a href="https://github.com/chancancode" rel="noopener noreferrer" target="_blank">Godfrey Chan (@chancancode)</a>, <a href="https://github.com/genisd" rel="noopener noreferrer" target="_blank">Daniel (@genisd)</a>, <a href="https://github.com/xg-wang" rel="noopener noreferrer" target="_blank">Thomas Wang (@xg-wang)</a>, <a href="https://github.com/amyrlam" rel="noopener noreferrer" target="_blank">Amy Lam (@amyrlam)</a>, <a href="https://github.com/NullVoxPopuli" rel="noopener noreferrer" target="_blank">@NullVoxPopuli</a>, <a href="https://github.com/jherdman" rel="noopener noreferrer" target="_blank">James Herdman (@jherdman)</a>, and <a href="https://github.com/bertdeblock" rel="noopener noreferrer" target="_blank">Bert De Block (@bertdeblock)</a> for their contributions to Ember and related repositories! 💖</p>
```

**Comment by hergaiety** [comment]
Skimmed and this is looking great~!

**Comment by amyrlam** [comment]
@gossi thank you very much for the submission here and on discord, much appreciated! sorry we didn't get to it, sort of getting back into the swing of things for the new year

let's push it to the next issue, let me make a draft and tag you post-publish of this issue tonite

(also, if you're ever up for submitting a short blurb for the times, please do!)

**Pull request merged**